sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/GenerateSchemaTask.kt

```diff
@@ -25,13 +26,14 @@ import java.sql.DriverManager
 import javax.inject.Inject
 
 @CacheableTask
-open class GenerateSchemaTask
-@Inject constructor(val workerExecutor: WorkerExecutor) : SourceTask() {
+abstract class GenerateSchemaTask : SourceTask() {
   @Suppress("unused") // Required to invalidate the task on version updates.
-  @Input fun pluginVersion() = VERSION
+  @Input val pluginVersion = VERSION
+
+  @get:Inject
+  abstract val workerExecutor: WorkerExecutor
 
   @get:OutputDirectory
-  @get:PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Wasn't this added for remote cache support?

Reply:
gradle told me it dont work with @OutputDirectory

if people want to contribute something back again i will accept